File: app/assets/javascripts/backbone/models/hood.js
App.Models.Hood = Backbone.Model.extend({
  urlRoot: "/hoods",

  assets: ["bike_shares", "charter_schools", "childcare_facilities", "embassies", "grocers", "libraries", "metros", "nightclubs", "pharmacies", "place_of_worships",  "police_stations"],

  initialize: function(){
    this.getAssets();
  },

  getAssets: function(asset){
    var self = this;
    $.getJSON("/hoods/" + self.attributes.id)
      .done(function(response){
        self.assets.forEach(function(asset){
          self.set(asset, response[asset]);
        });
      });
  }
});

File: app/models/hood.rb
class Hood < ActiveRecord::Base
  has_many :bike_shares
  has_many :charter_schools
  has_many :childcare_facilities
  has_many :embassies
  has_many :grocers
  has_many :libraries
  has_many :metros
  has_many :nightclubs
  has_many :pharmacies
  has_many :place_of_worships
  has_many :police_stations
end
